pkg/cmd/roachtest/tests/acceptance.go

```diff
@@ -76,6 +76,13 @@ func registerAcceptance(r registry.Registry) {
 				numNodes: 3,
 			},
 		},
+		registry.OwnerSQLFoundations: {
+			{
+				name:    "validate-system-schema-after-version-upgrade",
+				fn:      runValidateSystemSchemaAfterVersionUpgrade,
+				timeout: 30 * time.Minute,
```

Comment:
I think we need `defaultLeases: true` here.

Reply:
done